lib/elixir/lib/module.ex

```diff
@@ -695,6 +695,16 @@ defmodule Module do
   defp simplify_signature(other, acc) when is_binary(other),  do: autogenerated(acc, :binary)
   defp simplify_signature(_, acc),                            do: autogenerated(acc, :arg)
 
+  defp simplify_struct(module) when is_atom(module) do
```

Comment:
Since this function simplifies a *struct name* I would highly recommend to name it `simplify_struct_name/1`, as `simplify_struct` has a very large amount of possible meanings :)